guides/source/7_2_release_notes.md

```diff
@@ -21,6 +26,252 @@ guide.
+and service worker, which are served from `app/views/pwa` and can be dynamically rendered through ERB. Those files
+are mounted explicitly at the root with default routes in the generated routes file.
+
+For more information, see the [pull request adding the feature](https://github.com/rails/rails/pull/50528).
+
+### Add omakase RuboCop rules by default
+
+Rails applications now come with [RuboCop](https://rubocop.org/) configured with a set of rules from [rubocop-rails-omakase](https://github.com/rails/rubocop-rails-omakase) by default.
+
+Ruby is a beautifully expressive language that not only tolerates many different dialects, but celebrates their
+diversity. It was never meant as a language to be written exclusively in a single style across all libraries,
+frameworks, or applications. If you or your team has developed a particular house style that brings you joy,
+you should cherish that.
+
+This collection of RuboCop styles is for those who haven't committed to any specific dialect already. Who would just
+like to have a reasonable starting point, and who will benefit from some default rules to at least start a consistent
+approach to Ruby styling.
+
+These specific rules aren't right or wrong, but merely represent the idiosyncratic aesthetic sensibilities of Rails'
+creator. Use them whole, use them as a starting point, use them as inspiration, or however you see fit.
+
+### Add GitHub CI workflow by default to new applications
+
+Rails now adds a default GitHub CI workflow file to new applications. This will get especially newcomers off to a good
+start with automated scanning, linting, and testing. We find that a natural continuation for the modern age of what
+we've done since the start with unit tests.
+
+It's of course true that GitHub Actions are a commercial cloud product for private repositories after you've used the
+free tokens. But given the relationship with GitHub and Rails, the overwhelming default nature of the platform for
+newcomers, and the value of teaching newcomers good CI habits, we find this to be an acceptable trade-off.
+
+### Add Brakeman by default to new applications
+
+[Brakeman](https://brakemanscanner.org/) is a great way to prevent common security vulnerabilities in Rails from going
+into production.
+
+New application come with Brakeman installed and combined with the GitHub CI workflow, it will run automatically on
+every push.
+
+### Set a new default for the Puma thread count
+
+Rails changed the default number of threads in Puma from 5 to 3.
+
+Due to the nature of well-optimized Rails applications, with quick SQL queries and slow 3rd-party calls running via jobs,
+Ruby spends most of its time waiting for the Global VM Lock (GVL) to release when the thread count is too high.
+
+After careful consideration, investigation, and based on battle-tested experience from applications running in
+production, we decided that a default of 3 threads is a good balance between concurrency and performance.
```

Comment:
Would adding guidance on number of workers also make sense here? I've read through the associated issue. IIUC, the number of workers should equal the number of vCPUs, and the number of threads per worker should be between 1 (for well-optimized rails apps) and 3. Is that a fair summary?

Reply:
That probably should be discussed in the issue. For now number of threads is all we have.

Reply:
Yeah, that doesn't make sense to include such things in the release notes, it's more for the performance tuning guide.

It's all explained there.